packages/renderer/MdToHtml/rules/image.ts

```diff
@@ -11,9 +11,9 @@ function plugin(markdownIt: any, ruleOptions: RuleOptions) {
 		const token = tokens[idx];
 		const src = utils.getAttr(token.attrs, 'src');
 		const title = utils.getAttr(token.attrs, 'title');
+		const alt = token.content;
 
 		if (!Resource.isResourceUrl(src) || ruleOptions.plainResourceRendering) return defaultRender(tokens, idx, options, env, self);
-
```

Comment:
no whitespace change please

Reply:
ok got it. I will take care from now on.